TODO.md

```diff
@@ -0,0 +1,19 @@
+# TODO
+- upgrade to Svelte@4
+- migrate e2e/svelte to Vite (see e2e/svelte/README.md)
+- cleanup README
+- add eslint with plugins (svelte, import, promise)
+- use different config loader https://github.com/unjs/c12
+- use different build tool https://github.com/unjs/unbuild
```

Comment:
I like that this project is using Rollup now. It's so common in the Svelte ecosystem it makes it really easy to pick up and start working on the project. I've never used unbuild on the otherhand. I think there's also some possibility we could ship unbundled ESM and don't even need a build tool at all

Reply:
Im fine with rollup. The build tool is really just needed to convert ESM to CJS.